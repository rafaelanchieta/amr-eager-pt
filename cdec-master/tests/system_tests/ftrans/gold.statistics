-lm_nodes 13
-lm_edges 16
-lm_paths 4
