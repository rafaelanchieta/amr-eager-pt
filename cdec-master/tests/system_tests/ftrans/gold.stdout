0 ||| CB ||| Inv=1 F4=1 ||| 101
0 ||| BC ||| Mono=1 F3=0.4 ||| 6
0 ||| B C ||| Mono=1 F1=1 OtherFeat=1 F2=0.2 ||| 3
0 ||| C B ||| Inv=1 F2=0.2 F1=1 OtherFeat=1 ||| 2
